From db4fb4c0422aefaf4f76023c1f13ea440287f14a Mon Sep 17 00:00:00 2001
From: Minh Le <minh.le.aj@renesas.com>
Date: Tue, 27 May 2025 18:40:45 +0700
Subject: [PATCH 27/46] =?UTF-8?q?arm64:=20dts:=20renesas:=20ironhide:=20se?=
 =?UTF-8?q?t=20I2C0=20clock=20to=20100=E2=80=AFkHz?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Reduce I2C0 transfer rate to 100 kHz as a precaution.

On some boards, poor signal quality at higher speeds
may cause communication issues. This change ensures
stability even in those cases.

Signed-off-by: Minh Le <minh.le.aj@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a78000-ironhide.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a78000-ironhide.dts b/arch/arm64/boot/dts/renesas/r8a78000-ironhide.dts
index 115d7e285c41..b49bf3cec320 100644
--- a/arch/arm64/boot/dts/renesas/r8a78000-ironhide.dts
+++ b/arch/arm64/boot/dts/renesas/r8a78000-ironhide.dts
@@ -119,7 +119,7 @@ &i2c0 {
 	pinctrl-names = "default";
 
 	status = "okay";
-	clock-frequency = <400000>;
+	clock-frequency = <100000>;
 
 	eeprom@50 {
 		compatible = "rohm,br24g01", "atmel,24c01";
-- 
2.34.1

